CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Changelog - Fyne.io fyne-cross
 
-## 1.3.0 - Unreleased
+## Unreleased
+
+## 1.3.0 - 16 Jul 2022
 
 ### Added
 
 - Add support for web target #92
-- Add CI job to build calculator app #104 
+- Add CI job to build calculator app #104
```

Comment:
I am not so sure that we want CI stuff to be in the changelog. Not relevant for users

Reply:
That was actually from the previous release. I have update the changelog with v1.4.0

Reply:
I don't know what git is doing here, but this was part of the previous release change not this one. It feels incorrect to go back and remove things from the CHANGELOG after it was published.

Reply:
Ah, I see. Sorry. My GitHub page was messing with me and made it look like part of this release